libmambapy/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+libmambapy 0.18.2 (November 24, 2021)
+=====================================
+
+
```

Comment:
maybe it would be better to not add an empty section?

Reply:
@wolfv it would have been nice to modify the script to remove this artifact

the output name is also not necessary since the changelog is located in the sub-project directory

Reply:
we can fix that later down the line. I think it's OK to have it like this.